apps/sample-ui/tsconfig.json

```
@@ -1,5 +1,5 @@
 {
   "extends": "tsconfig/cra.json",
-  "include": ["**/*.ts", "**/*.tsx"],
+  "include": ["**/*.ts", "**/*.tsx", "src/.emv", "src/mocks/handlers.js", "src/mocks/browser.js"],
```

```suggestion
  "include": ["**/*.ts", "**/*.tsx", "src/.emv", "src/**/*.js"],
```
Probably fine like that

---

My bad, this was some boilerplate code...I changed those js files to ts and works well now :)